app/models/concerns/laboratory_date_validator.rb

```diff
@@ -8,5 +8,8 @@ def validate(record)
     year_start = Date.new(2020, 1, 1)
     validate_between_dates(record, :report, year_start, Time.now.to_date) if record.report_changed?
     validate_between_dates(record, :specimen_collection, year_start, Time.now.to_date) if record.specimen_collection_changed?
+    return unless record.report && record.specimen_collection && (record.specimen_collection > record.report)
+
+    record.errors.add(:specimen_collection, 'Report Date cannot be before Specimen Collection Date.')
```

Comment:
The error should be on `:report` and the message should be worded `"cannot be before Specimen Collection Date."` because that's what the error formatters expect grammatically.

Otherwise it will look like this
![image](https://user-images.githubusercontent.com/8235974/120731727-2fcb2f80-c4b2-11eb-9e46-56a020327cb3.png)